app/config/messaging.js

```diff
@@ -2,43 +2,43 @@ const Joi = require('joi')
 
 const msgTypePrefix = 'uk.gov.ffc.ahwr'
 
-const mqSchema = Joi.object({
+const schema = Joi.object({
   messageQueue: {
     host: Joi.string().required(),
     username: Joi.string(),
     password: Joi.string(),
-    useCredentialChain: Joi.bool().default(false),
+    useCredentialChain: Joi.bool(),
     appInsights: Joi.object()
   },
   applicationRequestQueue: {
-    address: process.env.APPLICATIONREQUEST_QUEUE_ADDRESS,
-    type: 'queue'
+    address: Joi.string().required(),
+    type: Joi.string().required().valid('queue')
   },
-  applicationRequestMsgType: `${msgTypePrefix}.app.request`,
   applicationResponseQueue: {
-    address: process.env.APPLICATIONRESPONSE_QUEUE_ADDRESS,
-    type: 'queue'
+    address: Joi.string().required(),
+    type: Joi.string().required().valid('queue')
   },
   eventQueue: {
-    address: process.env.EVENT_QUEUE_ADDRESS,
-    type: 'queue'
+    address: Joi.string().required(),
+    type: Joi.string().required().valid('queue')
   },
-  fetchApplicationRequestMsgType: `${msgTypePrefix}.fetch.app.request`
+  applicationRequestMsgType: Joi.string().required()
```

Comment:
Good spot on this, thanks for raising.

I'd removed `fetchApplicationRequestMsgType` because it wasn't actually exported. So naturally I assumed nothing was trying to import it. Turns out there was a method `getApplication` which did so. However, on further investigation it seems that nothing actually calls `getApplication`, and as far as I can tell [nothing ever did](https://github.com/DEFRA/ffc-ahwr-farmer-apply/commit/b803ff2d6a357e2c5b52cfdf314ddc5e9bd2c5a7)

So I've removed that too, in a new commit.